core/API/DataTableManipulator/ReportTotalsCalculator.php

```diff
@@ -126,6 +131,18 @@ protected function manipulateDataTable($dataTable)
             $clone->filter('AddColumnsProcessedMetrics', array($deleteRowsWithNoVisit = false));
         }
 
+        // remove processed metrics as they might haven been incorrectly summed up before
+        $clone->deleteMetadata(DataTablePostProcessor::PROCESSED_METRICS_COMPUTED_FLAG);
+        $processedMetrics = $clone->getMetadata(DataTable::EXTRA_PROCESSED_METRICS_METADATA_NAME);
+        if (!empty($processedMetrics) && is_array($processedMetrics)) {
+            foreach ($processedMetrics as $metric) {
+                if ($metric instanceof ProcessedMetric) {
+                    $metric = $metric->getName();
+                }
+                $clone->deleteColumn($metric);
+            }
+        }
```

Comment:
By moving the totals row it might now be the case that processed metrics are already calculated before. That has the effect, that the totals row calculation would simply sum up the values. As the values are then already present in the totals row, the processing triggered below wouldn't do anything, as the columns are already present.

Note: I also tried setting `skip` as aggregation operation for the processed metrics, hoping that would cause the columns to be missing in the result, but it turned out they are still created, but with `null` values. Therefor this can't be used, as the processing would still be skipped, as the column technically exists.